doc/attributes.md

```diff
@@ -67,3 +67,50 @@ message:
 
 When an attribute is used in Istio, its name is given along with a value. The value must be of the type declared by the corresponding descriptor. This
 type checking makes it possible for the Istio system to statically detect or prevent many configuration errors.
+
+# Standard Istio Attribute Vocabulary
+
+In a standard Istio deployment (using the Istio proxy), the system will produce the following set of attributes. This is considered the standard, or canonical, attribute set for Istio.
+
+| Name| Type| Description| Kubernetes Example| HTTP Header Name| Notes|
```

Comment:
Let's drop the Notes and HTTP Header Name sections for now, those need more thought.